stylesheet/main.css

```diff
@@ -15,12 +17,25 @@ a {
 
 header nav a {
     color: #5E5369;
+    display: block;
+    padding-top: 1em;
+    padding-bottom: 0.5em;
+    margin-left: 1em;
+    margin-right: 1em;
+    border-top: 4px solid transparent;
+}

```

Comment:
This rule applies to hyperlinks specifically in the header, so this is where you should be adding your declarations that apply to those elements.